src/main/webapp/profile.html

```diff
@@ -16,8 +16,28 @@
   </head>
   <body>
     <div id="content">
-      <!-- TODO: add navbar here. -->
-      <h1>Your Profile</h1>
+      <nav class="navbar navbar-expand-lg navbar-light bg-light">
```

Comment:
Duplicating this bar on every page is not going to be maintainable, because you will have to update this in lock step in all pages.

Not a problem in the prototype stage, but make a mental note to look into a templating engine for the real thing. A good templating engine with builtin support in app engine is JSP.

Here's the relevant documentation: https://cloud.google.com/appengine/docs/standard/java/config/webxml#jsps There are also tutorials available.